Makefile

```diff
@@ -99,18 +97,14 @@ doc:
 doc-open:
 	cargo doc --no-default-features --features esp32 --target=xtensa-esp32-espidf -Zbuild-std=std,panic_abort --open
 
-size:
-	find . -name "esp-build.map" -exec ${IDF_PATH}/tools/idf_size.py {} \;
-
-build-esp32-bin: build-esp32-ota
+build-esp32-bin:
 	cargo +esp espflash save-image \
 		--skip-update-check \
 		--package=micro-rdk-server \
-		--features=ota \
```

Comment:
ota is now a default on micro-rdk do we still need to have a ota feature in the micro-rdk-server crate or is it safe to remove?

Reply:
Safe to remove, I think, unless there is a use-case for disabling it, which I can't think of.